vulnfeeds/cmd/download-cves/main.go

```diff
@@ -74,26 +84,68 @@ func downloadCVE2(apiKey string, cvePath string) {
 	if err != nil {
 		log.Fatalf("Error reading response body: %+v", err)
 	}
-	var NVD2Data cves.NVDCVE2
+	var NVD2Data *cves.NVDCVE2
 	err = json.Unmarshal(body, &NVD2Data)
 	if err != nil {
 		log.Fatalf("Failed to decode NVD data: %+v", err)
 	}
-	Logger.Infof("%d results to download", NVD2Data.TotalResults)
-	// TODO: figure out how to paginate
-	file.Write(body)
+	Logger.Infof("At offset %d of %d total results", *NVD2Data.StartIndex, *NVD2Data.TotalResults)
+	return *NVD2Data
+}
+
+// Download all of the CVE data using the 2.0 API
+// See https://nvd.nist.gov/developers/vulnerabilities
+func downloadCVE2(APIKey string, CVEPath string) {
+	file, err := os.OpenFile(path.Join(CVEPath, "nvdcve-2.0.json.new"), os.O_CREATE|os.O_RDWR|os.O_TRUNC, 0644)
+	defer file.Close()
+	if err != nil { // There's an existing file, check if it matches server file
+		Logger.Fatalf("Something went wrong when creating/opening file: %+v", err)
+	}
+	completeNVDDataset := cves.NVDCVE2{}
```

Comment:
Since we are using just one field of NVDCVE2 (i.e. `Vulnerabilities`), might as well just make it an array of `Vulnerabilities`

Reply:
Ah, now I understand what you were saying earlier. Done in https://github.com/google/osv.dev/pull/773/commits/cb8262e93c0cd4d084f887e1effc050b285d9202